mesheryctl/internal/cli/root/environments/create_test.go

```diff
@@ -0,0 +1,170 @@
+package environments
+
+import (
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+
+	// "github.com/layer5io/meshkit/utils/kubernetes/describe"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateEnvironmentCmdValidation(t *testing.T) {
```

Comment:
We do not need to create a seperate `TestCreateEnvironmentCmdValidation` function to test for testing missing flag. `TestCreateEnvironmentCmd` should handle all the test cases

Reply:
I created two function because flag check test does not require to create a mock api response with fixture file.
Should I add a flag check boolean for testcases and merge the two test functions?

Reply:
Okay, got it now. Can you add a check and merge the functions? If it works, that should be a better approach.

Reply:
In the new commit i have merged the test without adding an extra check.

Reply:
Also i am struggling with assigning null values to flags in every new  test. The flag value for the older test gets copied in the new one. Can you guide me on this?